src/log/src/LogLine.tsx

```diff
@@ -9,27 +18,25 @@ export default defineComponent({
     }
   },
   setup (props) {
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    const { trimRef, highlightRef, languageRef, mergedHljsRef } = inject(
-      logInjectionKey
-    )!
+    const { trimRef, highlightRef, languageRef, mergedHljsRef } =
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      inject(logInjectionKey)!
     const selfRef = ref<HTMLElement | null>(null)
+    const mergedLineTrimRef = computed(() => {
+      return trimRef.value ? (props.line || '').trim() : props.line
```

Comment:
```suggestion
      return trimRef.value ? (props.line || '').trim() : props.line
```

`(props.line || '')` 没必要

Reply:
虽然之前也是我写的= =